build.assets/Dockerfile

```diff
@@ -182,6 +181,13 @@ RUN (curl -L https://github.com/bats-core/bats-core/archive/v1.2.1.tar.gz | tar
      cd bats-core-1.2.1 && ./install.sh /usr/local && cd .. && \
      rm -r bats-core-1.2.1)
 
+# Install shellcheck.
+RUN scversion='v0.8.0' && \
```

Comment:
Do we want this to be `latest` or `stable` ? So that we get any possible patch releases automatically?

Reply:
I initially had `stable`, but opted for named versions so we have more control over eventual breakages.